gcapi/client.py

```diff
@@ -845,25 +847,72 @@ def update_archive_item(
             )
         )
 
-    def create_display_sets_from_images(
-        self, *, reader_study: str, display_sets: List[Dict[str, list]]
+    def _fetch_interface(self, slug):
+        try:
+            interface = yield from self.__org_api_meta.interfaces.detail(
+                slug=slug
+            )
+            return interface
+        except ObjectNotFound as e:
+            raise ValueError(
+                f"{slug} is not an existing interface. "
+                f"Please provide one from this list: "
+                f"https://grand-challenge.org/components/interfaces/reader-studies/"
+            ) from e
+        return interface
+
+    def _validate_display_set_values(self, values, interfaces):
+        invalid_file_paths = {}
+        for slug, value in values:
+            if interfaces.get(slug):
+                interface = interfaces[slug]
```

Comment:
Shouldn't this raise a ValidationError? Right now someone could add twice the same interface.

Reply:
No, this goes over all display sets that are provided and is just to make sure we don't do a `GET` for the same interface for each display set.

Reply:
Ah ok, I see it now. Thanks for explaining!